x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -10002,51 +10002,6 @@
     "xpack.csp.cspPageTemplate.packageNotInstalled.pageTitle": "Installer l'intégration pour commencer",
     "xpack.csp.cspPageTemplate.packageNotInstalled.solutionNameLabel": "Niveau de sécurité du cloud",
     "xpack.csp.cspSettings.rules": "Règles de sécurité du CSP - ",
-    "xpack.csp.findings.agentLabel": "Agent",
```

Comment:
all changes to translations files are done by running `node scripts/i18n_check.js --fix ` and are needed to fix CI errors.